docs/configuration.md

```diff
@@ -61,3 +61,20 @@ spec:
 ```
 
 > Currently, you can set only 1 namespace to watch in this flag. See [this Kubernetes issue](https://github.com/kubernetes/contrib/issues/847) for more details.
+
+### Limiting External Namespaces
+
+Setting the `--restrict-scheme` boolean flag to `true` will enable the ALB controller to check configmap named `alb-ingress-controller-internet-facing-ingresses` for a list of approved ingresses before provisioning ALBs with an internet-facing scheme. Here is an example of that ConfigMap:
+
+```yaml
+apiVersion: v1
+data:
+ mynamespace: my-ingress-name
+ myothernamespace: my-other-ingress-name
+kind: ConfigMap
+metadata:
+  name: alb-ingress-controller-internet-facing-ingresses
+```
+
+
+That ConfigMap is kept in `default` if unspecified, but can moved to another with the `ALB_CONTROLLER_RESTRICT_SCHEME_CONFIG_NAMESPACE` environment variable.
```

Comment:
Perhaps mentions flag here, `--restrict-scheme-namespace` similar to how you've mentioned the flag `--restrict-scheme` above.